lib/style/deprecations.ex

```diff
@@ -58,6 +58,21 @@ defmodule Styler.Style.Deprecations do
     defp style({:|>, m, [lhs, {{:., _, [{_, _, [mod]}, :slice]} = f, funm, [{:.., _, [_, _]} = range]}]})
          when mod in [:Enum, :String],
          do: {:|>, m, [lhs, {f, funm, [add_step_to_decreasing_range(range)]}]}
+
+    # ~R is deprecated in favor of ~r
+    defp style({:sigil_R, m, args}), do: {:sigil_r, m, args}
+
+    # For a decreasing range, we must use Date.range/3 instead of Date.range/2
+    defp style({{:., _, [{:__aliases__, _, [:Date]}, :range]} = funm, dm, [first, last]} = block) do
```

Comment:
tyty, should the pipe version be hit as well? (technically this will trigger on `date |> Date.range(foo, bar)` _but_ since bar will never be a date your extracte_date_value will fail and it'll skip np np)

Reply:
I will add support for the pipe version, hold on :)

Reply:
Yes, I was implying that it was probably not worth adding another function head to handle the case of another date literal, but I ended up adding it because who knows is out there.